Medkit Resprite

Conflee opened the pull request:
## Short description
A resprite of all Medkits to revert cabinetification. 

## Why we need to add this
While I like the new cigarette box and ammo box sprites, Cabinetification did not work for Medkits or Cardboard Boxes. So, I am redoing both in parts. First up was Medkits since they came out the worst.

This resprite includes all Medkits: Brigmed Kube, Advanced, Brute, Burn, Combat, Normal, Oxygen, Radiation, and Toxin.

## Media (Video/Screenshots)
![image](https://github.com/user-attachments/assets/3cb33f79-1e9b-4f1b-8c65-a3c8f7236d1a)
Sprites in-game.

## Checks

- [X] I do not require assistance to complete the PR.
- [X] Before posting/requesting review of a PR, I have verified that the changes work.
- [X] I have added screenshots/videos of the changes, or this PR does not change in-game mechanics.
- [X] I affirm that my changes are licensed under the [Starlight Fork License](https://github.com/ss14Starlight/space-station-14/blob/Starlight/LICENSE-Starlight.TXT) and grant permission for use in this repository under its conditions.

**Changelog**

:cl: Conflee
- tweak: Resprite of all Medkits to a new style, undoing Cabinetification in the process

Conflee committed:
Delete Resources/Textures/Objects/Specific/Medical/firstaidkits.rsi/advkit-inhand-left.png

Conflee committed:
Delete Resources/Textures/Objects/Specific/Medical/firstaidkits.rsi/advkit-inhand-right.png

Conflee committed:
Delete Resources/Textures/Objects/Specific/Medical/firstaidkits.rsi/advkit.png

Conflee committed:
Delete Resources/Textures/Objects/Specific/Medical/firstaidkits.rsi/blackkit-inhand-left.png

Conflee committed:
Delete Resources/Textures/Objects/Specific/Medical/firstaidkits.rsi directory

Conflee committed:
Add files via upload

Conflee committed:
Delete Resources/Textures/_Starlight/Objects/Specific/Medical/firstaidkits.rsi directory

Conflee committed:
Add files via upload

Conflee committed:
Delete Resources/Prototypes/Entities/Objects/Specific/Medical/medkits.yml

Conflee committed:
Add files via upload

Rinary1 reviewed (changes requested):
huh? Why you change original files but also add new sprites in our folder?

Conflee commented:
> huh? Why you change original files but also add new sprites in our folder?

Dark had me do the same for the Job Bags so I just did it here too. Added the new sprites to both, changed the prototypes to use the ones in our folder. That way is Wiz changes them again upstream it won't overwrite ours.
